sidecar/src/figwheel_sidecar/components/figwheel_server.clj

```diff
@@ -232,8 +309,8 @@
      ;; users handler goes last
      (possible-endpoint resolved-ring-handler)
 
-     (handle-static-resources http-server-root)
-     (handle-index            http-server-root)
+     (handle-static-resources http-server-root supports-extended-attributes?)
```

Comment:
Do you prefer to calculate this once and pass it through to both handlers, or calculate it inside each handler?

Reply:
I'm thinking we skip the extended attributes and try to keep this brutally simple.

Reply:
less code less maintainance